### ISSUE TITLE
using maxNativeZoom with leaflet-deepzoom

### ISSUE THREAD

**Issue by jdbeaver** [alfarisi/leaflet-deepzoom#2]
I am having problems using maxNativeZoom and leaflet-deepzoom. I'm using the example supplied with leaflet-deepzoom and introducing maxNativeZoom.

Provided is the snippet of code that I have modifed:

```
var map = L.map('image2d', {
        maxZoom:16
 }).setView(new L.LatLng(0,0), 0);

map.setMaxBounds(new L.LatLngBounds(
      map.unproject([-100, (Math.pow(2, map.getMaxZoom()) * 128) + 100],   map.getMaxZoom()),
      map.unproject([Math.pow(2, map.getMaxZoom()) * 256, -100], map.getMaxZoom())
));

var tileLayerOptions = {
        width: 2400,
        height: 3000,
        tolerance: 0.8,
        maxZoom:16,
        maxNativeZoom:12
}
L.tileLayer.deepzoom('DeepZoomImage/hubble_files/', tileLayerOptions).addTo(map);
```

The issue is that tiles disappear after level 13. I've tried various suggestions for the same issue that others not using leaflet-deepzoom have had with no luck.

**Comment by jdbeaver** [comment]
Forked and made necessary changes to provide over-scaling on deep zoom tiles.

**Comment by leesei** [comment]
@jdbeaver care to submit a PR?
Or can I take your to to submit a PR to this repo?

**Comment by alfarisi** [comment]
solved in the latest version (2.0.0)
thank you
